test/check-global.js

```diff
@@ -0,0 +1,3 @@
+if (!global.globalSetup) {
```

Comment:
Maybe overkill to duplicate these files but wanted to prove the full ESM format case